src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SeparatorWrapCheckTest.java

```diff
@@ -91,7 +91,7 @@ public void testInvalidOption() throws Exception {
         try {
             final String[] expected = CommonUtils.EMPTY_STRING_ARRAY;
 
-            verify(checkConfig, getPath("InputSeparatorWrap.java"), expected);
+            verify(checkConfig, getPath("InputSeparatorWrapAfter3904.java"), expected);
```

Comment:
if i make different files and put those files in same folder so i will not be able to make all interfaces like foo,bar etc in every file